### ISSUE TITLE
Add a link to globalsign fork in the README

### ISSUE THREAD

**Issue by feliixx** [go-mgo/mgo#528]
This repository is no longer maintained, but people keep opening issues here. A link pointing to the fork maintained by **globalsign** ( [https://github.com/globalsign/mgo](https://github.com/globalsign/mgo) ) in the README  would be useful, could you consider adding one ? 

Thanks in advance, 
feliixx

**Comment by theodesp** [comment]
It's not that the project is not maintained. It's not actively maintained.
Please see https://github.com/go-mgo/mgo/issues/416

For now, the main focus of the project is stability, that's why you won't see risky commits being merged.

**Comment by feliixx** [comment]
@theodesp well issues/PR are left unanswered for more than a year now... People waste their time trying to contribute to a non actively-maintained project, while they could contribute directly to the maintained fork

**Comment by theodesp** [comment]
@feliixx I agree. But there is a fine line between acknowledging that the project is not maintained actively and not maintained at all. It is this the latter case then I agree its best if we reference a link to the active repo.

And one other thing. If you search for `go mongodb` or `go mongodb driver` on Google you will get only this project in the results. I think the lack of an official MongoDB driver may contribute to that state.
